docs/3____physical_signal_abstraction.adoc

```diff
@@ -44,9 +44,11 @@ This allows the FMU (or better the exporting tool) to balance the accuracy and p
  - While `aperiodic` Clocks allow very accurate network simulations, frequently entering `Event Mode` might reduce the network simulation speed.
  - Using `periodic` Clocks reduces the number of `Event Modes` required and might speeds up the simulation at the cost of simulation accuracy, because timing must be forced into a fixed grid and some intermediate values might not be transmitted.
  - One could use (structural) parameters to define the accuracy of `aperiodic` Clocks, allowing control of the simulation accuracy and performance with the same FMU.
- - The input Clocks shall be `triggered` Clocks.
+ - The input Clocks shall be `triggered` Clocks, if they are to be driven by the respective output Clock.
+ - Alternatively, input Clocks can be `periodic` Clocks matching the relevant output Clock, which causes them to be scheduled for the same time instances.
+   This allows for very efficient simulation in the case that there exists a more or less static communications schedule, and more detailed simulation of timing variation is not needed.
```

Comment:
```suggestion
   _[This may allow for more efficient simulation in the case that there exists a more or less static communications schedule, and more detailed simulation of timing variation is not needed.]_
```